code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -102,7 +102,7 @@
 	taste_description = "slime"
 
 /datum/reagent/vaccine/reaction_mob(mob/living/L, method=TOUCH, reac_volume)
-	if(islist(data) && (method == INGEST || method == INJECT))
+	if(islist(data) && method == INJECT)
```

Comment:
????

Reply:
save me from this torment, what did you do to the main branch when you merged that PR?